src/tasks/pair-liquidity-info-history-importer/pair-liquidity-info-history-importer.service.ts

```diff
@@ -224,7 +224,7 @@ export class PairLiquidityInfoHistoryImporterService {
     const pairBalances =
       await this.mdwClient.getContractBalancesAtMicroBlockHash(
         pairWithTokens.address as ContractAddress,
-        block.hash,
+        block.hash as MicroBlockHash,
```

Comment:
shouldn't this already be a MicroBlockHash as the type `MicroBlock` has this already defined is already defined?

Reply:
You are absolutely right. Good catch!

Reply:
I reverted the change.